layers/+tools/terraform/packages.el

```diff
@@ -1,6 +1,6 @@
 ;;; packages.el --- terraform Layer packages File for Spacemacs
```

Comment:
Please also add a layer dependency on lsp in layers.el when the backend is set to lsp, to automatically load lsp when someone has set the backend respectively. Again please refer to the python layer for a good example how to do this.